### ISSUE TITLE
stopped working with VSCode 1.40

### ISSUE THREAD

**Issue by StefanoCecere** [fabiospampinato/vscode-projects-plus#54]
i've been using Projects+ since more than 1 year. and today i discover the first time it doesn't work at all: no projects, no bottom bar menu.

from the console i could read:

```
Activating extension 'fabiospampinato.vscode-projects-plus' failed: 'isExtensible' on proxy: trap result does not reflect extensibility of proxy target (which is 'true').
_logMessageInConsole @ abstractExtensionService.ts:396
```

```
Error: command 'projects.refresh' not found
    at u._tryExecuteCommand (commandService.ts:80)
    at commandService.ts:73
    at async o.run (actions.ts:190)
onDidNotificationChange @ notificationsAlerts.ts:38
```

**Comment by StefanoCecere** [comment]
PS: i solved temporarily going back to VSCode 1.39
i read many problems with the new 1.40 which updates Electron to v6.0

**Comment by fabiospampinato** [comment]
Duplicate of https://github.com/fabiospampinato/vscode-projects-plus/issues/51
